docs/datatypes/mesh/index.md

```diff
@@ -2576,6 +2576,16 @@ init();</script></body>
 </html>" width="100%" height="500px" style="border:none;"></iframe
 
 
+To display a colored point cloud, store the corresponding colors in the `.tensor` of a chunk `Document` with the name tag `'point_cloud_colors'`. The colors have to be of shape (n_samples, 3) or (n_samples, 4).
+
+```python
```

Comment:
is there a way we can load both color and data from some files ?

Reply:
okay I see and in practice how would I load the color with trimesh ? How do I get this colors array ?

Reply:
okay I see makes sense then